frontend/yarn.lock

```diff
@@ -33,17 +33,203 @@
   version "0.7.0"
   resolved "https://registry.yarnpkg.com/@sindresorhus/is/-/is-0.7.0.tgz#9a06f4f137ee84d7df0460c1fdb1135ffa6c50fd"
 
+"@types/c3@^0.6.0":
```

Comment:
A ton of dependencies are getting added here... Per @jeff-phillips-18, patternfly-react should be doing tree shaking. We should verify we're only loading what we're using.

Reply:
Best I can tell, all of these dependencies are getting included in the vendor bundle (c3.js, etc.) for overview. It's almost doubling the size from our main vendor bundle. We're only using a couple components. (This is running in development mode. I'll also check the built image.)

@jeff-phillips-18 any ideas?

![project overview okd 2018-08-24 09-45-44](https://user-images.githubusercontent.com/1167259/44587989-898b6080-a782-11e8-8d5e-36daf7073ee6.png)

<img width="903" alt="project overview okd 2018-08-24 09-42-50" src="https://user-images.githubusercontent.com/1167259/44587939-5fd23980-a782-11e8-9928-78c7628d9493.png">

Reply:
Development mode does not tree shake

Reply:
I was seeing it in the built image, too, but I think I didn't correctly build it. Thanks @jeff-phillips-18, I think we're OK then.

Reply:
OK guys. Now that I built it correctly, looks like we're good. Sorry for the false alarm!

![project overview 2018-08-24 10-13-15](https://user-images.githubusercontent.com/1167259/44589367-69f63700-a786-11e8-85cc-3ed3462ed7f5.png)

Reply:
Curious about that. Is there any concern we will forget about the tree shaking in dev mode & accidentally ship a large bundle to prod in the future?  Or is there any risk that dev mode might load something that won't exist later, meaning we won't see the error now?  Just curious about the possibilities here.

Reply:
The tree shaking stuff should just work... And with code splitting across routes, it should protect us as well.

Reply:
The "Performance" e2e tests run against the production build, so we are covered from regressions there.